chunkflow/chunk/base.py

```diff
@@ -378,17 +378,27 @@ def crop_margin(self, margin_size: tuple = None, output_bbox: Bbox=None):
             assert output_bbox is not None
             return self.cutout(output_bbox.to_slices())
     
-    def threshold(self, threshold: float):
-        seg = self > threshold
+    def threshold(self,
+                  threshold: float,
+                  remap_high: int = None,
```

Comment:
I am not really an expert of python type annotation, does the line actually pass the check? since you set `remap_high` to `None` which is not `int`?

Reply:
I'm also new to Python type hinting, and you're right, [PEP484](https://peps.python.org/pep-0484/#union-types) says it's not recommended. It should be `remap_high: Optional[int] = None` instead. Will modify it.

Your meticulous review is truly useful! I'm really enjoying it.